src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -191,6 +192,9 @@ namespace Microsoft::Console::VirtualTerminal
         }
 
     private:
+        static constexpr VTParameter defaultParameter{};
+        static constexpr std::span defaultParameters{ &defaultParameter, 1 };

```

Comment:
Is `defaultParameters` actually used anywhere?

Reply:
Yep, while `at()`'ing outside the range. Or atleast that's what I intended.

Reply:
Oops, you asked `defaultParameters` and not `defaultParmeter`, sorry.

I should remove it.